locales/en/app.ftl

```diff
@@ -798,3 +798,23 @@ vpn-promo-cta = Get { -brand-mozilla-vpn }
 
 vpn-promo-headline-new = Save 50% with a full year subscription
 vpn-promo-copy-new = Protect your online data—and choose a VPN subscription plan that works for you.
+
+## VPN educational/promotional banner.  HTML tags should not be translated, e.g. `<em>`
+# user's location is populated dynamically by 3rd-party service, eg: "Your location: Los Angeles, CA".  The 3rd-party service will need to provide localization.
+vpn-banner-location = Your location:
+vpn-banner-protect-yourself-with-vpn = <em>Protect yourself</em> with { -brand-mozilla-vpn }.
+vpn-banner-protected-with-vpn = <em>Protected</em> with { -brand-mozilla-vpn }.
+vpn-banner-title-1 = You’re protected — thanks for using { -brand-mozilla-vpn }.
+vpn-banner-title-2 = Your location can be tracked if you don’t use a VPN.
+vpn-banner-subtitle-2 = Protect your location and browse securely in 3 steps
+vpn-banner-status-protected = Current status: <em>Protected ✓</em>
+vpn-banner-status-not-protected = Current status: <em>Not protected ⚠</em>
+# user's IP address is determined dynamically, eg: "IP address: 192.168.1.1"
+vpn-banner-ip-address = IP address:
+vpn-banner-step-1 = Subscribe to { -brand-mozilla-vpn }
+vpn-banner-step-2 = Select a VPN location
+vpn-banner-step-3 = Activate VPN and browse securely
+vpn-banner-cta = Get { -brand-mozilla-vpn }
+# button to expand or close panel
```

Comment:
This is current a single string comment, applying to the next string, it should be a group comment.

Reply:
updated.